.github/workflows/build.yml

```diff
@@ -75,7 +64,58 @@ jobs:
         id: docker_build
         uses: docker/build-push-action@v2
         with:
+          load: true
           push: ${{ github.event_name != 'pull_request' }}
-          tags: k8ssandra/reaper-operator:${{ steps.vars.outputs.sha_short }}
+          tags: k8ssandra/reaper-operator:${{ steps.vars.outputs.sha_short }}, k8ssandra/reaper-operator:latest
           cache-from: type=local,src=/tmp/.buildx-cache
           cache-to: type=local,dest=/tmp/.buildx-cache
+      - name: Export reaper-operator image
+        run: |
+          docker save k8ssandra/reaper-operator:latest --output=/tmp/k8ssandra-reaper-operator.tar
+      - name: Upload reaper-operator image
+        uses: actions/upload-artifact@v2
+        with:
+          name: reaper-operator
+          path: /tmp/k8ssandra-reaper-operator.tar
+  run_e2e_test:
```

Comment:
Are we pushing the image to docker hub before running the e2e tests? If so would it better to push only after the tests pass?

Reply:
We're not pushing in PRs at all.